indico/modules/events/registration/fields/simple.py

```diff
@@ -330,3 +341,20 @@ def _to_machine_date(date):
 
 def _to_date(date):
     return datetime.strptime(date, '%Y-%m-%d').date()
+
+
+class MultiChoiceField(ChoiceBaseField):
+    name = 'multi_choice'
+    wtf_field_class = wtforms.SelectMultipleField
+
+    @property
+    def wtf_field_kwargs(self):
+        return {'choices': [(id, caption) for id, caption in self.form_item.data['captions'].iteritems()]}
```

Comment:
also, this can be in the case class - it works for single choice as well as for multi choice